solutions/cses-1676.mdx

```diff
@@ -56,9 +64,61 @@ int main () {
 
 </CPPSection>
 
-<JavaSection>
+<PySection>
+
+```python
+import sys
+input = sys.stdin.readline
+
+# BeginCodeSnip{Disjoint Set Union}
```

Comment:
you can remove the spaces

Reply:
just the one in the beginning? or all the spaces?

Reply:
all codesnip spaces